.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: c
+
+services:
+        - docker
+
+before_install:
+        - docker pull ghdl/ext:latest
+
+os:
+        - linux
+
+script:
+        - docker run --volume `pwd`:/build --rm -t ghdl/ext:latest bash -c "cd /build && make && make -j$(nproc) check"
```

Comment:
You can use `--workdir`, instead of `cd`:

```bash
docker run --rm -t -v `pwd`:/build -w /build ghdl/ext:latest bash -c "make && make -j$(nproc) check"
```

Moreover, according to https://github.com/ghdl/ghdl/commit/44079dc238cadc72c4b65452f195150ff1e81f83:

```bash
docker run --rm -t -v `pwd`:/build -w /build ghdl/ext:latest bash -c "make GNATMAKE='gnatmake -j'$(nproc) && make -j$(nproc) check"
```